Tasks/VsTest/vstest.ts

```diff
@@ -91,8 +92,13 @@ function getTestAssemblies(): string[] {
         tl.debug('Search directory empty, defaulting to ' + vstestConfig.testDropLocation);
     }
 
-    tl.debug('Searching for test assemblies in: ' + vstestConfig.testDropLocation);
-    return tl.findMatch(vstestConfig.testDropLocation, vstestConfig.sourceFilter);
+    if (pathExistsAsDirectory(vstestConfig.testDropLocation)) {
```

Comment:
Can this check be added in taskInputParser only?

Reply:
Is it a warning or error?

Reply:
This check doesn't prints anything by itself. We will print a warning in the else clause.